src/style/5_components/_components.GearListItem.scss

```
@@ -0,0 +1,28 @@
+.gear-list-item {
+	font-size: $font-size-sm;
+	line-height: 1.4;
+
+	.img-thumbnail {
```

`<ImageThumbnail class="gear-list-item__img-thumbnail">`

```suggestion
	__img-thumbnail {
```